src/main/java/tds/exam/results/mappers/TestMapper.java

```diff
@@ -21,13 +31,16 @@
             .findFirst().get();
 
         test.setBankKey(bankKey);
-//        test.setHandScoreProject(assessment.getHandScoredProject());
-//        test.setContract(assessment.getContract());
+        test.setHandScoreProject(new Long(assessment.getHandScoreProjectId()));
```

Comment:
Don't think it matters in the grand scheme of things but I usually use Long.valueOf instead.  This [answer](http://stackoverflow.com/questions/13145267/difference-between-long-valueofjava-lang-string-and-new-longjava-lang-string) has a good explanation